webpack/assets/javascripts/react_app/components/ReportTemplate/Inputs/ReportDateTime.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
it's nested under ReportTemplate, but it's generic for other template types right? should it be it's own directory

Reply:
now it is generic